pycbc/fft/fftw.py

```diff
@@ -153,7 +157,7 @@ def set_threads_backend(backend=None):
             raise RuntimeError("Could not initialize FFTW threading backend {0}".format(backend))
     else:
         # Note that we pop() from the end, so 'openmp' is the first thing tried
-        _backend_list = ['unthreaded','pthreads','openmp']
+        _backend_list = ['unthreaded','openmp', 'pthreads']
```

Comment:
Please also change the comment line immediately above to indicate that now 'pthreads' is the first thing tried, or we'll get very confused later.